From 309480007a49cf3a90a48bb3bab0b1149a70f061 Mon Sep 17 00:00:00 2001
From: mstickel <michael.stickel@baslerweb.com>
Date: Fri, 25 Oct 2019 15:48:48 +0200
Subject: [PATCH] add CONFIG_DEFAULT_FDT_FILE to imx8mq_evk in u-boot

---
 include/configs/imx8mq_evk.h | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/include/configs/imx8mq_evk.h b/include/configs/imx8mq_evk.h
index 647850cd2e..c41630e985 100644
--- a/include/configs/imx8mq_evk.h
+++ b/include/configs/imx8mq_evk.h
@@ -13,6 +13,10 @@
 
 #define CONFIG_SYS_BOOTM_LEN		(32 * SZ_1M)
 
+#ifndef CONFIG_DEFAULT_FDT_FILE
+#define CONFIG_DEFAULT_FDT_FILE "imx8mq-evk.dtb"
+#endif
+
 #define CONFIG_SPL_MAX_SIZE		(148 * 1024)
 #define CONFIG_SYS_MONITOR_LEN		(512 * 1024)
 #define CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_USE_SECTOR
@@ -105,7 +109,7 @@
 	"fdt_addr=0x43000000\0"			\
 	"fdt_high=0xffffffffffffffff\0"		\
 	"boot_fdt=try\0" \
-	"fdtfile=imx8mq-evk.dtb\0" \
+	"fdtfile=" CONFIG_DEFAULT_FDT_FILE "\0" \
 	"bootm_size=0x10000000\0" \
 	"mmcdev="__stringify(CONFIG_SYS_MMC_ENV_DEV)"\0" \
 	"mmcpart=" __stringify(CONFIG_SYS_MMC_IMG_LOAD_PART) "\0" \
-- 
2.25.1

